src/app/pages/OriginsLaunchpad/components/PreviousSalesCardSection/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { PreviousSalesRow, PreviousSalesRowWrapper } from './styled';
+import { PreviousSalesCard } from '../PreviousSalesCard/index';
+import imgGenesis from 'assets/origins_launchpad/GEN-NFT-COMMUNITY.svg';
+import imgOrigins from 'assets/origins_launchpad/GURU_BADGER_NFT.svg';
+
+export const PreviousSalesCardSection: React.FC = () => (
+  <PreviousSalesRowWrapper>
+    <PreviousSalesRow>
+      <PreviousSalesCard
+        saleName="SOV Origins Sale"
+        saleAllocation="2,000,000 SOV"
```

Comment:
Numbers should use `Number.toLocaleString` prototype or to `toNumberFormat` helper for displaying, hardcoding format will make only couple of regions to understand what is the actual number here.

Reply:
And currently displaying number formats are also different for each cards, here it's 2,000,000 and in genesis sale card it's 2641946.1868

Reply:
Good catch, I'll fix the numbers and regarding the menu - it's more the question for @t-z-u.

Reply:
The numbers format issue is fixed.